changelog/3664.deprecation.rst

```diff
@@ -0,0 +1 @@
+Deprecate applying a mark to a fixture
```

Comment:
Similar to comment above, would be good to add some remediation details here, e.g. `Applying a mark to a fixture function is deprecated. Doing so has no effect. If you want to apply a mark to all functions which request the fixture, use XYZ instead.`